bots/schemas/createBot.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Bot",
+  "description": "A ButtCloud Pub",
+  "type": "object",
+  "properties": {
+    "userId": {
```

Comment:
the `userId` will be added by the server in a hook, so it's not part of the schema that the user submits when creating a pub.